django_react_proj/users-fe/src/index.js

```diff
@@ -6,9 +6,9 @@ import reportWebVitals from './reportWebVitals';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <React.Fragment>
```

Comment:
If you don't need to use strict mode (which is debatable) you can use plain  `<App />` w/o wrapping it with fragment